frontend/src/components/navbar/index.tsx

```diff
@@ -1,108 +1,49 @@
-
-    };
-    const userMenuProps = { userSignin, handleProfileMenuOpen }
-
-    const location = useLocation();
-    const isHomePage = location.pathname === "/";
-
-    return (
-        <div id="header-container">
-            <AppBar position="fixed">
-                <Toolbar>
-                    <Link to="/" style={ linkToLanding }>
-                        <Typography
-                            variant="h6"
-                            noWrap
-                            component="div"
-                            sx={ { display: { xs: "none", sm: "block" } } }
-                        >
-                            E-commerce
-                        </Typography>
-                    </Link>
-                    { isHomePage && (
-                        <Search>
-                            <SearchIconWrapper>
-                                <SearchIcon />
-                            </SearchIconWrapper>
-                            <StyledInputBase
-                                placeholder="Search…"
-                                inputProps={ { "aria-label": "search" } }
-                            />
-                        </Search>
-                    ) }
-                    <Box sx={ { flexGrow: 1 } } />
-                    <Box sx={ { display: { xs: "none", md: "flex" } } }>
-                        <ShoppingCart />
-                        <UserMenu { ...userMenuProps } />
-                    </Box>
-                    <Box sx={ { display: { xs: "flex", md: "none" } } }>
-                        <IconButton
-                            size="large"
-                            aria-label="show more"
-                            aria-controls={ mobileMenuId }
-                            aria-haspopup="true"
-                            onClick={ handleMobileMenuOpen }
-                            color="inherit"
-                        >
-                            <MoreIcon />
-                        </IconButton>
-                    </Box>
-                </Toolbar>
-            </AppBar>
-            <MobileMenu { ...MobileMenuProps } />
-            <ProfileMenu { ...menuProfileProps } />
-        </div>
-    );
-}
+import UserOptionsMobileMenu from "./UserOptionsMobileMenu";
+import UserOptionsMenu from "./UserOptionsMenu";
+import ProductSearch from "./ProductSearch";
+import IsUserOptionsMenuOpen from "./IsUserOptionsMenuOpen";
+import IsUserOptionsMobileMenuOpen from "./IsUserOptionsMobileMenuOpen";
+import { HOME_PATH } from "../constants";
+
+const Navbar = () => (
+    <div id="navbar-container">
+        <AppBar position="fixed">
+            <Toolbar>
+                <Link to={ HOME_PATH } style={ landingLinkStyles }>
+                    <Typography
+                        variant="h6"
+                        noWrap
+                        component="div"
+                        sx={ { display: { ...linkDisplayStyles } } }
+                    >
+                        E-commerce
```

Comment:
it's time to give it a better name, no? 😄

Reply:
True, how about MarketHive ?